# 1.already localize with the help of human being.
# 2.ready to navigation.
int32 map_id
string map_name
int32 nav_flag
basic_msgs/gridpose gridpose

---
float64[] data
string errormsg
bool  successed